interfaces/builtin/camera.go

```diff
@@ -48,12 +51,13 @@ const cameraConnectedPlugAppArmor = `
 /run/udev/data/c81:[0-9]* r, # video4linux (/dev/video*, etc)
 /run/udev/data/+usb:* r,
 /sys/class/video4linux/ r,
-/sys/devices/pci**/usb*/**/video4linux/** r,
+/sys/devices/**/video4linux/** r,
 `
 
 var cameraConnectedPlugUDev = []string{
 	`KERNEL=="video[0-9]*"`,
 	`KERNEL=="vchiq"`,
+	`KERNEL=="io[0-9]*"`,
```

Comment:
I don't think this rule is specific enough, for example this will also match /dev/io, which from all my googling I don't think will ever in practice be used on linux, but on freeBSD it is a special device node: https://manpages.debian.org/buster/freebsd-manpages/io.4freebsd.en.html. 

There is also this kernel patch (proposed but never merged afaict): https://lwn.net/Articles/479817/ that suggests to me that maybe someday /dev/io could be something in Linux.

Also, I think that this rule will match other subpath devices such as `/dev/something/io`, but admittedly I cannot find any such devices, the closest I could find to this on https://www.kernel.org/doc/Documentation/admin-guide/devices.txt is `/dev/vx/iod`, but I still would feel more comfortable if this rule was more specific given that "io" is so unspecific. 

I tried and failed to find out any kind of documentation about where this kernel driver comes from, though it seems it is packaged in Arch Linux perhaps: https://aur.archlinux.org/pkgbase/decklink/?setlang=hr&comments=all&O=150&PP=10? I think before merging this we need to do more investigation about this kernel module, as if it is more generic like /dev/uinput it should not be permitted in a non-superprivileged interface like camera, and